lldb/include/lldb/lldb-enumerations.h

```diff
@@ -1305,6 +1305,13 @@ enum CompletionType {
   eTerminatorCompletion = (1ul << 27)
 };
 
+/// Specifies if children need to be re-computed
+/// after a call to \ref SyntheticChildrenFrontEnd::Update.
+enum class ChildCacheState {
+  eRefetch = 0, ///< Children need to be recomputed dynamically.
+  eReuse = 1,   ///< Children will never change and don't need to be recomputed.
```

Comment:
is "will never change" true in all cases? Or should it be "did not change"?

Reply:
After thinking about this, I believe you are correct, it probably means "you can re-use what I gave the last time you called update"

Reply:
fixed, thanks!